src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -938,8 +938,7 @@ def get_store_info(self) -> ServerModel:
     # ------
     # Stacks
     # ------
-
-    @track(AnalyticsEvent.REGISTERED_STACK)
+    @track(AnalyticsEvent.REGISTERED_STACK, v2=True)
```

Comment:
There are around 22 different events being tracked in this file, but you only picked a handful to use for v2 tracking. What about the other ones ?

Reply:
We had a discussion with the team and decided on a subset of events that we would track as they gave us the most insight into the usage of ZenML. Events such as `COPIED_STACK` or `SET_STACK` were deemed uninformative.

Reply:
Just realized that the comment was on the `sql_zen_store` not on the `cli` functionalities. However, the same idea applies here as well. We were aiming to track the registration of critical entities such as stack, components and workspaces, however, their `update` and `deletion` processes or any processes of additional entities such as team and roles were deemed uninformative for the current version.